test/v200/utils/api/test_utils.go

```diff
@@ -0,0 +1,173 @@
+package api
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"testing"
+
+	common "github.com/devfile/api/v2/test/v200/utils/common"
```

Comment:
already is common, no need to alias

Reply:
I plan in my next pull request to make this commonUtils.

Reply:
Fixed in new commit